Resources/Prototypes/whitelists.yml

```diff
@@ -0,0 +1,38 @@
+﻿- type: PlayerConnectionWhitelist
+  id: mrpWhitelist
+  conditions:
+    - !type:ConditionManualBlacklist # Blacklisted people are banned from playing MRP but can still play LRP
+      breakIfConditionSuccess: false
+      breakIfConditionFail: true
+    - !type:ConditionNotes
+      includeExpired: false
+      minimumSeverity: 3 # High
+      minimumNotes: 1
+      range: 30 # 30 days
+    - !type:ConditionNotes
+      includeExpired: false
+      minimumSeverity: 2 # Medium
+      minimumNotes: 1
+      range: 14 # 14 Days
+    - !type:ConditionNotes
+      includeExpired: false
+      minimumSeverity: 1 # Low
+      minimumNotes: 3
+      range: 14 # 14 Days
```

Comment:
You are going to need a way to be able to re-use the notes when evaluating these 3 conditions. You're not getting away with calling `GetAllAdminRemarks` **three** times per connection attempt.